4.7.4/config/test.ttl

```diff
@@ -0,0 +1,21 @@
+@prefix :      <http://base/#> .
```

Comment:
For configuring inferencing, see:
https://github.com/fcrepo4-exts/fcrepo4-vagrant-base-box/blob/master/config/test.ttl